util/Migrator/DbScripts/2021-04-16_00_OrganizationUser_DeleteMany.sql

```diff
@@ -0,0 +1,194 @@
+-- Create sproc to bump the revision date of a batch of users
+IF OBJECT_ID('[dbo].[User_BumpAccountRevisionDateByOrganizationUserIds]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[User_BumpAccountRevisionDateByOrganizationUserIds]
+END
+GO
+
+CREATE PROCEDURE [dbo].[User_BumpAccountRevisionDateByOrganizationUserIds]
+    @OrganizationUserIds [dbo].[GuidIdArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
+        OU.UserId
+    INTO
+        #UserIds
+    FROM
+        [dbo].[OrganizationUser] OU
+    INNER JOIN
+        @OrganizationUserIds OUIds ON OUIds.Id = OU.Id
+    WHERE
+        OU.[Status] = 2 -- Confirmed
+
+    UPDATE
+        U
+    SET
+        U.[AccountRevisionDate] = GETUTCDATE()
+    FROM
+        [dbo].[User] U
+    INNER JOIN
+        #UserIds ON U.[Id] = #UserIds.[UserId]
+END
+GO
+
+-- Create TwoGuidIdArray Type
+IF NOT EXISTS (
+    SELECT
+        *
+    FROM
+        sys.types
+    WHERE
+            [Name] = 'TwoGuidIdArray' AND
+        is_user_defined = 1
+)
+CREATE TYPE [dbo].[TwoGuidIdArray] AS TABLE (
```

Comment:
Same here, I don't see this type file in the Sql project files under User Defined Types: https://github.com/bitwarden/server/tree/master/src/Sql/dbo/User%20Defined%20Types

Reply:
gotta change the branch: https://github.com/bitwarden/server/tree/support-large-organization-sync/src/Sql/dbo/User%20Defined%20Types

Sneaking up fixes for an earlier PR merge (#1287) since this is going to the same feature branch